### DISCUSSION TITLE
[bug] broken theme

### DISCUSSION THREAD

**Discussion by mateusfg7** [adityatelange/hugo-PaperMod#117] [Q&A] [answered]
I installed the theme and configured, when I run `hudo server` everything works as it should
![image](https://user-images.githubusercontent.com/40613276/101545578-386cfe00-3986-11eb-857a-b40cbce82a12.png)

but when I run `hugo` to generate the static website, it looks like this
![image](https://user-images.githubusercontent.com/40613276/101545832-9ac5fe80-3986-11eb-9a2f-3d6b74952666.png)

**Comment by kdkasad** [comment]
Can you post your `config.toml`/`config.yml`

**Comment by adityatelange** [comment]
@mateusfg7 https://adityatelange.github.io/hugo-PaperMod/posts/papermod/papermod-how-to/#failed-to-find-a-valid-digest-in-the-integrity-attribute-for-resource--

**Comment by mateusfg7** [comment, marked as answer]
I just run the command `hugo -b [my website]` and all works well.
I saw the file `public/index.html` and the assets was using `www.example.org/`, with this command the assets now is using my website source.
